auth/src/charm.py

```diff
@@ -117,15 +117,52 @@ def backend_grpc_endpoint(self) -> Optional[Endpoint]:
     ##################
     # EVENT HANDLERS #
     ##################
+    def _tls_is_consistent(self, remote_insecure: bool) -> bool:
```

Comment:
```suggestion
    def _tls_required(self, remote_insecure: bool) -> bool:
```
Feel free to ignore, but `_tls_is_consistent` sounds a bit unclear to me.